ush/gfs_bfr2gpk.sh

```diff
@@ -21,14 +21,14 @@ source "${USHgfs}/preamble.sh"
 cd "${DATA}" || exit 2
 
 #  Set output directory:
-if [[ ! -d "${COM_ATMOS_GEMPAK}" ]]; then mkdir -p "${COM_ATMOS_GEMPAK}"; fi
+if [[ ! -d "${COMOUT_ATMOS_GEMPAK}" ]]; then mkdir -p "${COMOUT_ATMOS_GEMPAK}"; fi
 
 outfilbase=gfs_${PDY}${cyc}
 
 #  Get the list of individual station files.
 
 date
-cat "${COM_ATMOS_BUFR}/bufr."*".${PDY}${cyc}" > bufr.combined
+cat "${COMIN_ATMOS_BUFR}/bufr."*".${PDY}${cyc}" > bufr.combined
```

Comment:
@KateFriedman-NOAA Please change ${COMIN_ATMOS_BUFR} to ${COMOUT_ATMOS_BUFR}

Reply:
@BoCui-NOAA corrected @ f03d65a2e, thanks!

Reply:
Thanks for the updates. The bufr package including the code ownership look good to me.